crates/ruff_python_codegen/src/generator.rs

```diff
@@ -100,6 +105,13 @@ impl<'a> Generator<'a> {
         }
     }
 
+    /// Force quote style for generated source code.
+    #[must_use]
+    pub fn with_quote(mut self, quote: Quote) -> Self {
```

Comment:
Same as on the field documentation
```suggestion
    pub fn with_preferred_quote(mut self, quote: Quote) -> Self {
```